### ISSUE TITLE
Подобрать/реализовать шаблон (тему) сайта с верхним и боковыми меню

### ISSUE THREAD

**Issue by tsamsonov** [cartomsu/site#5]
Нужен такой шаблон сайта, в котором навигация будет трехуровневой: 

1. Основной раздел выбирается сверху (есть сейчас)
2. Подраздел выбирается слева (нету)
3. Оглавление выбранного подраздела показывается справа (есть сейчас)

При выборе основного раздела отображается какая-то общая страничка про него, а слева можно выбирать уже подразделы. Это решит проблему "скучного" вида страниц разделов, на которых есть только перечень подразделов. Также навигация станет более удобной. Сейчас чтобы выбрать другой подраздел, нужно вернуться назад (на уровень раздела).

Задача не срочная, сейчас в приоритете наполнение сайта контентом. Но в перспективе (в следующем году) хорошо бы ее решить.

**Comment by gtitov** [comment]
А как это будет на мобильных устройствах? Места слева уже не хватит. Может, наверх выносить выбор подраздела?

**Comment by tsamsonov** [comment]
Да, должна быть наверное какая-то адаптивная компоновка. -- <br />Отправлено из мобильной Яндекс.Почты: http://m.ya.ru/ymail28.11.2022, 13:12, "gtitov" ***@***.***>:
А как это будет на мобильных устройствах? Места слева уже не хватит. Может, наверх выносить выбор подраздела?

**Comment by gtitov** [comment]
Обновили дизайн
